.github/workflows/test.yml

```
@@ -21,3 +21,22 @@ jobs:
 
       - name: Test packages
         run: yarn test
+
+  e2e-tests:
+    name: Run e2e tests
```

[Minor] I think this is shortened as:

```suggestion
    name: Run E2E tests
```